sam-app/hello_world/app.py

```diff
@@ -23,3 +167,20 @@ def lambda_handler(event, context):
             'location': ip.text.replace('\n', ''),
         })
     }
+
+def lambda_handler(event, context):
+    """Handler function for the metricLogger Lambda function.
+
+    Parameters:
+        event (dict): the user input
+
+    Returns:
+        (str) message upon successful execution
+    """
+
+    validatorMessage = validator(event)
+    if validatorMessage != "Validated":
+        return validatorMessage
+    message = formMessage(event)
+    #metricLogger(message)
```

Comment:
If you call validator in your test instead of lambda_handler, then you can uncomment this.

Reply:
You don't need to alter functional code for unit tests. You can mock the boto3 client in your unit test so it doesn't actually make a call to CW Metrics.